chef/cookbooks/barclamp/libraries/barclamp_library.rb

```diff
@@ -35,6 +35,7 @@ def self.get_network_by_type(node, type)
               return Network.new(net, data, intf, interface_list)
             end
           end
+          return nil
```

Comment:
It is not, at least it wasn't when I played with that.
And current (master) code has it as well, see https://github.com/crowbar/crowbar-core/commit/0fa9984c9355c79f652d19a993c039ee27b90291